docs/process-definition/branches.md

```diff
@@ -33,10 +33,19 @@ There is only ever 1 `main` and 1 `incubation` branch. Feature branches start wi
 
 Read more on git tags & releases in our [release documentation].
 
-Understanding the commit history on a branch is important. Therefore when merging a branch into another, your PR branch must follow the pattern `merge-<source branch nam>`.
+## Merging upstream branches 
+
+Understanding the commit history on an upstream branch is important. Therefore when merging an upstream branch into another, your PR branch must follow the pattern `merge-<source branch nam>`.
 
 For example when merging `f/some-feature` into `incubation`, name your branch `merge-f/some-feature`. This will result in a commit message on the `incubation` branch of `Merge pull request # from <username>/merge-f/some-feature` when the PR is merged.
 
+Use the following steps to create a PR when merging upstream branches:
+
+- `git checkout -b merge-<source-branch> <target-branch>`
+- `git pull --no-rebase upstream <source-branch>`
+- Resolve all conflicts then post a PR.
+- Apply `approved` and `lgtm` labels to the PR once all checks pass.
```

Comment:
Explicitly call out this doesn't apply to merging to `main` please -- we need to be careful about that merge PR -- which I imagine can always just be `f/branch` to `main` directly and thus maybe doesn't quite follow this.